catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Node.js

```diff
@@ -0,0 +1,273 @@
+            this.organizationInfo.populateFromModel(this.get('RegistryObjectList').Organization, this.descriptors);
+
+            this.contactInfo = new Segment.Segment({
+                segmentName: 'Contacts',
+                multiValued: true,
+                segmentType: 'Person',
+                associationModel: this.associationModel
+            });
+            this.contactInfo.constructTitle = FieldDescriptors.constructPersonNameTitle;
+            this.contactInfo.populateFromModel(this.get('RegistryObjectList').Person, this.descriptors);
+
+            this.contentInfo = new Segment.Segment({
+                segmentName: 'Content Collections',
+                multiValued: true,
+                segmentType: 'Content',
+                associationModel: this.associationModel
+            });
+            this.contentInfo.constructTitle = FieldDescriptors.constructNameTitle;
+            var extrinsics = this.get('RegistryObjectList').ExtrinsicObject;
+            var contentOnly = _.without(extrinsics, _.findWhere(extrinsics, {objectType: 'urn:registry:federation:node'}));
+            this.contentInfo.populateFromModel(contentOnly, this.descriptors);
+
+            this.topLevelSegment.set('segments', new Backbone.Collection([this.generalInfo, this.serviceInfo, this.organizationInfo, this.contactInfo, this.contentInfo]));
+            this.associationModel.populateFromModel(this.get('RegistryObjectList').Association);
+        },
+        saveData: function () {
+            this.generalInfo.saveData();
+            this.serviceInfo.saveData();
+            this.organizationInfo.saveData();
+            this.contactInfo.saveData();
+            this.contentInfo.saveData();
+            var model = this;
+            model.get('RegistryObjectList').ExtrinsicObject = [this.generalInfo.dataModel[0]];
+            _.each(this.contentInfo.dataModel, function (content) {
+                model.get('RegistryObjectList').ExtrinsicObject.push(content);
+            });
+            this.associationModel.saveData();
+        },
+        validate: function(){
+            var errors = [];
+            this.appendErrors(errors, this.generalInfo.validate());
+            this.appendErrors(errors, this.serviceInfo.validate());
+            this.appendErrors(errors, this.organizationInfo.validate());
+            this.appendErrors(errors, this.contactInfo.validate());
+            this.appendErrors(errors, this.contentInfo.validate());
+            if(errors.length > 0){
+                return errors;
+            }
+            //no errors save data to dataModel
+            this.saveData();
+        },
+        appendErrors: function(errorArray,errors){
+          if(errors){
+              _.each(errors, function(error){
+                 errorArray.push(error);
+              });
+          }
+        },
+        sync: function () {
+
+            var deferred = $.Deferred();
```

Comment:
For some reason I wasn't returning the deferred like i was suppose to for the sync function. I am leaving it in now that it is the methods return value.